src/stats.rs

```diff
@@ -1,13 +1,21 @@
 use arc_swap::ArcSwap;
+use cadence;
+use cadence::{
+    prelude::*, BufferedUdpMetricSink, BufferedUnixMetricSink, MetricBuilder, NopMetricSink,
+    QueuingMetricSink, StatsdClient,
+};
 /// Statistics and reporting.
 use log::{error, info, trace};
 use once_cell::sync::Lazy;
 use parking_lot::Mutex;
 use std::collections::HashMap;
+use std::env;
+use std::net::UdpSocket;
```

Comment:
Should we be using Tokio sockets instead?

Reply:
None of this stuff is running within a tokio task (it's in it's own thread) so not sure what it gives us on top of the standard one, will need to look into it

Reply:
Cadence explicitly expects `std::net::UdpSocket` so can't use tokio one